students/KellyM/Lesson2/lesson 2 - series.py

```diff
@@ -0,0 +1,48 @@
+def fib(n):
+    """ This function calculates a Fibonacci Series, which is a series that
+        starts with 0 and 1, then calculates the next number by adding the
+        previous two numbers."""
+    if n == 1:
+        return 1
+    elif n==2:
+        return 1
+    else:
+        return (fib(n-1) + fib(n-2))
```

Comment:
Nice work identifying the recursive solution to this problem! FYI the outer parentheses are not necessary in the return statement